about.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Event</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Secular+One&family=Tajawal:wght@400;500;700;800;900&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <nav class="menuTop topMob">
+        <ul class="topMenuList topMenuListMob">
+          <li><a id="fb" href="#"><img class="sm" src="img/fb.svg" alt=""></a></li>
+          <li><a id="twitter" href="#"><img class="sm" src="img/twitter.svg" alt=""></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </nav>
+      <nav class="sideMargin mobMenu menuDesktop">
+        <a href="index.html"><img id="logo" src="img/logo.svg" alt="logo"></a>
+        <img id="hmbrg" src="img/hmbrg.svg" alt="">
```

Comment:
- [ ] I kindly suggest that you add a descriptive value to your alt attribute because the alt attribute is meant for adding a text description to the image. Please do the same for all the images on this HTML file.

:heavy_check_mark: Right
`<img id="hmbrg" src="img/hmbrg.svg" alt="hamburger logo">`